README.md

```diff
@@ -48,3 +48,4 @@ To configure individual rules:
 - [effector/no-useless-methods](/rules/no-useless-methods/no-useless-methods.md)
 - [effector/no-ambiguity-target](/rules/no-ambiguity-target/no-ambiguity-target.md)
 - [effector/prefer-sample-over-forward-with-mapping](/rules/prefer-sample-over-forward-with-mapping/prefer-sample-over-forward-with-mapping.md)
+- [effector/@typescript-no-watch](/rules/@typescript-no-watch/@typescript-no-watch.md)
```

Comment:
Maybe we ought to omit `typescript` from the name? What if in future someone'll find a way to improve it in JS?

Reply:
I'll fix it after I make the rest of the changes.